docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,132 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
```

Comment:
@kasiafi .. we hope to get the doc PR merged before the release on Friday and then rebase and just link here..

Reply:
Let's just link to #1839. The other issue should just be closed and all information added to the first one.

Reply:
@mosabua I closed https://github.com/trinodb/trino/issues/11937, and moved all the information to https://github.com/trinodb/trino/issues/1839. Please remove the link.
The documentation is updated and ready for your review.

Reply:
Okay ... removed link